src/meta/app_env_validator.cpp

```diff
@@ -158,9 +179,23 @@ void app_env_validator::register_all_validators()
          std::bind(&check_rocksdb_iteration, std::placeholders::_1, std::placeholders::_2)},
         {replica_envs::ROCKSDB_BLOCK_CACHE_ENABLED,
          std::bind(&check_bool_value, std::placeholders::_1, std::placeholders::_2)},
+        {replica_envs::READ_QPS_THROTTLING,
+         std::bind(&check_throttling, std::placeholders::_1, std::placeholders::_2)},
+        {replica_envs::READ_SIZE_THROTTLING,
+         std::bind(&utils::token_bucket_throttling_controller::validate,
+                   std::placeholders::_1,
+                   std::placeholders::_2)},
+        {replica_envs::SPLIT_VALIDATE_PARTITION_HASH,
+         std::bind(&check_bool_value, std::placeholders::_1, std::placeholders::_2)},
+        {replica_envs::USER_SPECIFIED_COMPACTION, nullptr},
+        {replica_envs::BACKUP_REQUEST_QPS_THROTTLING,
+         std::bind(&check_throttling, std::placeholders::_1, std::placeholders::_2)},
+        {replica_envs::ROCKSDB_ALLOW_INGEST_BEHIND,
+         std::bind(&check_bool_value, std::placeholders::_1, std::placeholders::_2)},
```

Comment:
Is this related to this patch? If not, better to seperate it to another PR, it would be much easier to track the relevant changes and review the code.

Reply:
I just move the method position, these are commented `no implement check` in the past: https://github.com/XiaoMi/rdsn/blob/v2.3.0/src/meta/app_env_validator.cpp#L170